### ISSUE TITLE
Kind suggestion.  to quicken adoption rate.. in-sdk in code annotations are VERY sparse..

### ISSUE THREAD

**Issue by johnda98** [hashgraph/hedera-hcs-sxc-java#431]
**Problem**
Unless a Dev is a super advanced java / Beans person..   adoption rate is slowed.

**Solution**
Mirror the annotation in the code.. like we see in the HCS simple example..  TONNES more in line code comments..  in fact have MORE examples - as this is the way non-polyglot Devs can quickly  juimp into syntax and lexicon of advanced Java and absorb the repo.

**Comment by gregscullard** [comment]
Hi John, thank you for the comment. The settlement demo is indeed complex (it's almost a working product in its own right) and not the easiest on ramp. Thinking about a simpler demo to illustrate the architecture and serve as a simple foundation to start building from, although a little more featured than the `simplemessagedemo` which is very very basic.

**Comment by johnda98** [comment]
thought this was close...  but hey.. yes I have a few jnr Devs(humans).. who have to get up to speed.. examples are great.. but comments are sparse inline code..  and worded descriptions of how the SDK is structures ie what a client is.. who holds a SCs paying account ie the contracts Hbar bal ..  etc..  how biginteger = uint256  etc etc ..    examples of higher vol fileservice storage ie larger files. .. I have to give them my code to deploy json abi's greater than the single chunk limit.. ie to use append etc.        IMPORTANTLY a simpler example of a full REACT or Vue DApp w spring backend and or even some rest controllers hitting hedera wrappers..    or how a topic listener is built in a spring backend

**Comment by johnda98** [comment]
How about a document on how a transaction is built .. ie tranbuilder   as a superclass .. and how hedera exceptions work.. what they are ..  types.. network, runtime - status.. etc.

ok.. all good ! :-)

**Comment by johnda98** [comment]
Its how the SDK maps to the principal component objects of a ledger function..  And a Public Ledger with a 1MB state store for contracts.. is useless in reality ..   HH will be just a crypto HCS machine without smart contracts as they are defined to be EXACTLY .. on the ledger ie VM held immutable.. SCs via HCS are junk.. not even BFT let alone ABFT.   EVEN if the user has to PAY in bars for more than 1MB of state store (to compensate the nodes to maintain TPS) .. then do so..    Truth has a price and users are willing to pay it even for a slightly lower speed of consensus, pref to ABFT.

**Comment by gregscullard** [comment]
Hi John, these seem to be sdk related issues (or both sdk and sxc ?). 

Envoyé depuis mon iPhone/Sent from my iPhone

**Comment by johnda98** [comment]
G .. whatever is good..  I just made a comment as suggestion.. this was closed by Zoperado .. then reopened 10mins ago.. I got a ghub notification.. so I decided to elaborate.  you guys can close it again..  I thing the suggestions are digested.. so all is good mate.   My CA client is all SCs .. I just build a DApp hitting a bunch.. stuffed it on GCP.. works well.      your demos are top box..  its just that 2 of my team are not springbooters..
